docs/user/release_notes.rst

```diff
@@ -34,6 +34,8 @@ Backwards incompatible changes
 * The ``bodhi-monitor-composes`` script has been removed (:issue:`2171`).
 * The stacks feature has been removed (:issue:`2241`).
 * The ``bodhi-manage-releases`` script has been removed (:issue:`2420`).
+* Support for anonymous comments was dropped. As a result, the ``anonymous`` field on the Comment
+  object was removed and comments query API parameter ``anonymous`` was droped (:issue:`2700`).
```

Comment:
We should also add a note about pillow, liberation-mono-fonts, and cryptography not being required dependencies anymore.